tests/functional/testOnboarding.js

```diff
@@ -369,4 +396,366 @@ describe('Onboarding', function() {
     assert.equal(await vpn.getQueryProperty(queries.screenOnboarding.DATA_CHECKBOX_ROW, 'isChecked'), 'true');
     assert.equal(await vpn.getSetting('onboardingDataCollectionEnabled'), true);
   });
-});
+
+  it('Onboarding started event is recorded', async () => {
```

Comment:
nit: Add all of the telemetry test cases inside a describe with a title like: 'onbording telemetry tests'. I've done this in other implementation, it's useful to do `npm run test -- --grep 'telemetry'` and just run the telemetry related tests.